eslint.config.cjs

```diff
@@ -1,50 +1,194 @@
-const js = require("@eslint/js");
-const ts = require("@typescript-eslint/eslint-plugin");
-const tsParser = require("@typescript-eslint/parser");
-
+/* eslint-disable unicorn/no-useless-fallback-in-spread */
+/* eslint-disable no-undef */
 //-----------------------------------------------------------------------------
 // NOTE: The following plugins have been removed from the package.json file
 // eslint-plugin-oclif linter repo has been archived
 // eslint-config-oclif-typescript linter repo has been archived
 
-// Export the final config
+// -----------------------------------------------------------------------------
+// Eslint packages used:
+// "eslint" –> The core ESLint package. It is the base linter.
+// NOTE: Styles have moved from eslint to @stylistic/eslint-plugin (js and ts)
+//       Built-in stylistic rules for JavaScript and Typescript
+//       We are still using legacy config, so we need to install v3.x with
+//       npm i -D @stylistic/eslint-plugin@3
```

Comment:
```suggestion
// TODO:    npm i -D @stylistic/eslint-plugin@3
```

Reply:
Is this something to do in the short term? Should we make it into an issue in the repo?